File: Day-2/Q5.cpp
#include <iostream>
using namespace std;

int sumDigits(long long n){
    int sum = 0;
    n = llabs(n);
    while(n > 0){
        sum += n % 10;
        n /= 10;
    }
    return sum;
}

int main(){
    int n;
    cout << "Enter a Number : ";
    cin >> n;
    cout << "Sum of Digits = " << sumDigits(n);
    return 0;
}

File: Day-2/Q5.java
import java.util.Scanner;

public class Q5 {
    public static int sumDigits(long n){
        int sum = 0;
        n = Math.abs(n);
        while(n > 0){
            sum += n % 10;
            n /= 10;
        }
        return sum;
    }

    public  static void main(String[] args){
        try(Scanner s = new Scanner(System.in)){
            System.out.print("Enter the Number : ");
            long n = s.nextLong();
            System.out.println("Sum of Digits = " + sumDigits(n));
        }
    }
}